projects/components/src/table/controls/table-controls-api.ts

```diff
@@ -1,58 +1,77 @@
 import { Dictionary } from '@hypertrace/common';
-import { SelectOption } from '../../select/select-option';
+import { FilterOperator } from '../../filtering/filter/filter-operators';
 import { TableFilter } from '../table-api';
 
-export interface SelectControl {
-  placeholder?: string;
-  default?: SelectOption<TableControlOption>;
-  options: SelectOption<TableControlOption>[];
+export const enum TableControlOptionType {
+  Filter = 'filter',
+  Property = 'property',
```

Comment:
What is a property control option? any visual example?

Reply:
Please see [this](https://github.com/hypertrace/hypertrace-ui/pull/823/files#r626981848) comment below.